code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -73,6 +73,18 @@
 	maxbodytemp = T0C + 40
 	held_state = "spacecat"
 
+/mob/living/simple_animal/pet/cat/breadcat
```

Comment:
might wanna make this a subtype of the cak

Reply:
not needed since he wouldn't want to retain the life functionality.

Do require the sample to return though.

Reply:
Don't see the sample return.

see https://github.com/tgstation/tgstation/blob/fd4ae63846450f16db1581776f6b2d1f2cc24da9/code/modules/mob/living/simple_animal/friendly/cat.dm#L98-L99

but it needs to be done with your path.